src/main/java/io/github/brassmc/brassloader/gui/InformationPanel.java

```diff
@@ -0,0 +1,25 @@
+package io.github.brassmc.brassloader.gui;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import io.github.brassmc.brassloader.boot.discovery.ModDiscovery;
+
+import javax.annotation.Nonnull;
+import java.util.HashMap;
+import java.util.Map;
+
+public class InformationPanel {
+
+    static final Map<String, Renderer> PANELS = new HashMap<>();
+
+    public static void createPanel(@Nonnull String modid, Renderer renderer) {
```

Comment:
This method does not create a panel, it registers one. 
```suggestion
    public static void registerPanel(@Nonnull String modid, Renderer renderer) {
```